mtp_bank_admin/apps/bank_admin/views.py

```diff
@@ -63,3 +65,11 @@ def download_bank_statement(request, receipt_date):
     })
 
     return response
+
+class Q_And_A_View(TemplateView):
```

Comment:
Python code style is underscores pretty much everywhere except class names, which should be camel case i.e. QAndAView